diff --git a/cmake/find/FindVulkanSDK.cmake b/cmake/find/FindVulkanSDK.cmake
index b0fe473..b4e6040 100644
--- a/cmake/find/FindVulkanSDK.cmake
+++ b/cmake/find/FindVulkanSDK.cmake
@@ -128,7 +128,7 @@ if (VULKANSDK_ROOT_DIR)
 
   if (UNIX)
     unset(VULKAN_LIB)
-    find_library(VULKAN_LIB NAMES vulkan HINTS "$ENV{VULKAN_SDK}/lib")
+    find_library(VULKAN_LIB NAMES libvulkan.so.1.3.204 HINTS "$ENV{VULKAN_SDK}/lib")
     get_filename_component(VULKAN_LIB_DIR ${VULKAN_LIB} DIRECTORY)
     find_library(VULKANSDK_SHADERC_LIB "libshaderc_combined.a" HINTS ${VULKAN_LIB_DIR})
     find_file(GLSLANGVALIDATOR VULKANSDK_ROOT_DIR "glslangValidator" HINTS ${VULKANSDK_ROOT_DIR}"../bin/glslangValidator")
@@ -175,7 +175,6 @@ SET(VULKANSDK_SHADERC_LIB ${VULKANSDK_SHADERC_LIB}          CACHE PATH "Path to
 find_package_handle_standard_args(VulkanSDK DEFAULT_MSG
     VULKANSDK_INCLUDE_DIR
     VULKAN_LIB
-    VULKANSDK_SHADERC_LIB
     GLSLANGVALIDATOR
 )
 
